app/home/view.html

```diff
@@ -1,3 +1,14 @@
+<div ng-if="$ctrl.sliders && $ctrl.sliders.length > 0">
```

Comment:
Please refer to the component section in `HOW TO GUIDE` and put this slider component in a separated folder. Thank you.